code/modules/events/immovable_rod.dm

```diff
@@ -47,6 +47,9 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 		qdel(src)
 	return ..()
 
+/obj/effect/immovablerod/ex_act(test)
+	return 0
+
 /obj/effect/immovablerod/Bump(atom/clong)
 	playsound(src, 'sound/effects/bang.ogg', 50, 1)

```

Comment:
can this be slowed down or limited to every second or third bump or something

It's really annoying to get 15 clangs before the rod goes out of your vision and it spams output